tests/e2e/default/reports/search-reports.wdio-spec.js

```diff
@@ -1,114 +1,73 @@
-      _id: sittuHospital._id,
-      parent: sittuHospital.parent
-    }
+    parent: { _id: sittuHospital._id, parent: sittuHospital.parent },
   });
-const potuPerson = personFactory.build(
-  {
+  const potuPerson = personFactory.build({
     name: 'Potu',
     patient_id: 'potu-patient',
-    parent: {
-      _id: potuHospital._id,
-      parent: potuHospital.parent
-    }
+    parent: { _id: potuHospital._id, parent: potuHospital.parent },
   });
 
-const PHONE_NO = '1234340002';
-
-const REPORT1 = {
-  _id: 'REF_REF_V1',
-  form: 'RR',
-  type: 'data_record',
-  from: PHONE_NO,
-  fields: {
-    patient_id: sittuPerson.patient_id
-  },
-  sms_message: {
-    message_id: 23,
-    from: PHONE_NO,
-    message: `1!RR!${sittuPerson.patient_id}`,
-    form: 'RR',
-    locale: 'en'
-  },
-  reported_date: moment().subtract(10, 'minutes').valueOf()
-};
+  const reports = [
+    smsPregnancyFactory.pregnancy().build({ fields: { patient_id: sittuPerson.patient_id } }),
+    smsPregnancyFactory.pregnancy().build({ fields: { patient_id: potuPerson.patient_id } }),
+    pregnancyFactory.build({ fields: { patient_id: sittuPerson.patient_id, case_id: 'case-12' } }),
+    pregnancyFactory.build({ fields: { patient_id: potuPerson.patient_id, case_id: 'case-12' } }),
+  ];
+  let reportDocs;
 
-const REPORT2 = {
-  _id: 'REF_REF_V2',
-  form: 'RR',
-  type: 'data_record',
-  from: PHONE_NO,
-  fields: {
-    patient_id: potuPerson.patient_id
-  },
-  sms_message: {
-    message_id: 23,
-    from: PHONE_NO,
-    message: `1!RR!${potuPerson.patient_id}`,
-    form: 'RR',
-    locale: 'en'
-  },
-  reported_date: moment().subtract(10, 'minutes').valueOf()
-};
-
-const docs = [...places.values(), sittuHospital, sittuPerson, potuHospital, potuPerson, REPORT1, REPORT2];
-
-describe('Test Reports Search Functionality', async () => {
   before(async () => {
-    await utils.saveDocs(docs);
+    await utils.saveDocs([ sittuHospital, sittuPerson, potuHospital, potuPerson ]);
+    reportDocs = await utils.saveDocs(reports);
     await loginPage.cookieLogin();
-    await commonPage.goToReports();
   });
 
-  it('search by NON empty string should display results with contains match and then clears', async () => {
-    // Waiting for initial load
-    await reportsPage.getAllReportsText();
+  it('should return results matching the search term and then return all data when clearing search', async () => {
+    await commonPage.goToReports();
+    // Asserting first load reports
+    expect((await reportsPage.reportsListDetails()).length).to.equal(4);
 
-    // Searching by keyword
     await searchPage.performSearch('sittu');
-    let allLHSContacts = await reportsPage.getAllReportsText();
-    expect(allLHSContacts.sort()).to.deep.equal([sittuPerson.name]);
+    await commonPage.waitForLoaders();
+    expect((await reportsPage.reportsListDetails()).length).to.equal(2);
+    expect(await (await reportsPage.reportByUUID(reportDocs[0].id)).isDisplayed()).to.be.true;
```

Comment:
I think we should denominate these reports somehow, instead of referring to them by index. 
Something like: 
```
const [smsSittu, smsPottu, pregnancySittu, pregnancyPottu] = await utils.saveDocs(reports);
...
expect(await (await reportsPage.reportByUUID(smsSittu.id)).isDisplayed()).to.be.true;
```